src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -49,20 +52,50 @@
 
 __all__ = [
     "load_pretrained_quantization",
+    "load_transforms",
     "apply_quantization_config",
     "apply_quantization_status",
     "find_name_or_class_matches",
     "expand_target_names",
     "is_target",
+    "process_transforms_config",
 ]
 
 from compressed_tensors.quantization.utils.helpers import is_module_quantized
-from compressed_tensors.utils.safetensors_load import get_quantization_state_dict
+from compressed_tensors.utils.safetensors_load import (
+    get_quantization_state_dict,
+    get_weight_mappings,
+)
+from safetensors import safe_open
 
 
 _LOGGER = logging.getLogger(__name__)
 
 
+def load_transforms(model: Module, model_name_or_path: str):
+    model_path = get_safetensors_folder(model_name_or_path)
+    weight_mappings = get_weight_mappings(model_path)
+
+    state_dict = {}
+    for weight_name, safe_path in weight_mappings.items():
+        if "transform" in weight_name:
+            with safe_open(safe_path, framework="pt", device="cpu") as f:
+                state_dict[weight_name] = f.get_tensor(weight_name)
+
+    for name, submodule in iter_named_leaf_modules(model):
+        transform_data = getattr(submodule, "transform_data", None)
```

Comment:
> TransformData includes a dictionary of all the transforms-relevant runtime data, and is attached to the layer as "transform_data"

To me, it seems like this information is essentially duplicating the information of the scheme? Why not attach the scheme to the module, rather than creating a new abstraction?